owncloudApp/src/main/java/com/owncloud/android/ui/activity/FileDisplayActivity.kt

```diff
@@ -294,7 +299,7 @@ class FileDisplayActivity : FileActivity(), FileFragment.ContainerActivity, OnEn
     private fun initFragmentsWithFile() {
         if (account != null && file != null) {
             /// First fragment
-            listOfFilesFragment?.listDirectory(currentDir)
+            listOfFilesFragment?.listDirectory(file)
```

Comment:
Is this a needed change? `currentDir` will use `file` if it's a folder anyway, right? Just curious

Reply:
it's no needed!